### ISSUE TITLE
translate place types when possible

### ISSUE THREAD

**Issue by eroux** [BuddhistDigitalResourceCenter/owl-schema#13]
I think it would be more neutral and logical to have place types in English instead of Tibetan Wylie (ex: `bdr:PlaceTypeMchodRten` -> `bdr:PlaceTypeStupa`). Of course some specific to Tibet may not be translatable...

**Comment by eroux** [comment]
there also needs to be a label on the place types. I think this starts to be important, it doesn't make sense for a Chinese dataset to use individuals with a Tibetan identifier... English is a more neutral language

**Comment by xristy** [comment]
I agree that labels  in various languages need to be added to the various pre-defined individuals in the ontology. In fact we are needing to add labels and comments on many ontology elements

It is also reasonable to change the URI for the individuals to forms that may be more readily recognized by diverse users
